src/observability/observability.star

```diff
@@ -45,6 +48,13 @@ def expose_metrics_port(ports, port_id=METRICS_PORT_ID, port_num=METRICS_PORT_NU
         ethereum_package_shared_utils.HTTP_APPLICATION_PROTOCOL,
     )
 
+def expose_pprof_port(ports, port_id=PPROF_PORT_ID, port_num=PPROF_PORT_NUM):
```

Comment:
just reuse `expose_metrics_port`?

Reply:
Yeah I was thinking that. But I would probably rename it then, or at least keep the two functions and defer to a common helper.

Reply:
[66d7612](https://github.com/ethpandaops/optimism-package/pull/352/commits/66d76128a7710cc194f343079c8291518252cf42)